CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 CHANGELOG
 =========
 
+0.1.0 (2016-XX-XX)
```

Comment:
Should this be set to a date or is it handled automatically when this gets released?